tests/unit/amo/pages/TestGuides.js

```diff
@@ -45,7 +45,7 @@ describe(__filename, () => {
     );
   };
 
-  it('fetches the add-ons for a guide page', () => {
+  it('fetches the addons for a guide page', () => {
```

Comment:
I believe we write `add-ons` in writing, and `addon` is code because we can't use an hyphen in variable names.

Reply:
I think I have seen it both ways in tests. I can change this back and change the other tests here tho.

Reply:
All editorial content must use the hyphen (*add-ons*) but that's the only hard rule we live by. Anything goes in the code 😁